docs/topics/native/native-objc-interop.md

```diff
@@ -133,6 +133,22 @@ Note that the opposite reversed translation is not implemented yet:
 Swift/Objective-C error-throwing methods aren't imported to Kotlin as
 exception-throwing.
 
+### Suspending functions
+
+> The support for calling `suspend` functions from Swift code as `async` is [Experimental](components-stability.md).
+> It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We would appreciate hearing your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issues/KT-47610).
+>
+{type="warning"}
+
+Kotlin's [suspending functions](coroutines-basics.md) (`suspend`) are presented in the generated Objective-C headers as
+functions with callbacks, or `completionHandler` in the Swift/Objective-C terminology.
+
+Swift 5.5 is bringing the [support for concurrency at the language level](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html)
+with the `async`/`await` mechanism. Kotlin's `suspend` functions are available for calling from Swift 5.5 and newer as
```

Comment:
The documentation doesn't mention that one can call Kotlin `suspend` function as a regular function with completion handler from Swift, in all Swift versions, including 5.5.
Without that, one can read this paragraph wrong.

Reply:
Thanks, I rewrote this part for better clarity.